doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -401,6 +414,31 @@ Displaying information about notations
    If on, parentheses are printed even if implied by associativity and precedence
    Default is off.
 
+
+.. cmd:: Print Grammar @ident
+
+   Shows the grammar for the nonterminal :token:`ident`, which must be one of the following:
+
+   - `constr` - for :token:`term`\s
+   - `pattern` - for :token:`pattern`\s
+   - `tactic` - for currently-defined tactic notations, :token:`tactic`\s and tacticals with
+   - `vernac` - for :token:`command`\s
+
+   The first three of these give the precedence and associativity for each construct.
+   For example, these lines printed by `Print Grammar tactic` indicates that the `try` construct
+   is at level 3 and right-associative.  `SELF` represents the `tactic_expr` nonterminal
```

Comment:
Shouldn't it be:
```suggestion
   is at level 3 and right-associative.  `SELF` represents the :n:`@ltac_expr` nonterminal
```

Reply:
This was intentional because `Print Grammar tactic` shows `Entry tactic:tactic_expr is` as the nonterminal name.

I tweaked the wording here:

```
   - `tactic` - for currently-defined tactic notations, :token:`tactic`\s and tacticals
     (corresponding to :token:`ltac_expr` in the documentation).
```